kernel/src/fs/file_handle.rs

```diff
@@ -116,6 +117,17 @@ pub trait FileLike: Pollable + Send + Sync + Any {
     fn as_socket(&self) -> Option<&dyn Socket> {
         None
     }
+
+    fn read_info(&self, offset: usize, writer: &mut VmWriter) -> Result<usize> {
```

Comment:
The method name may not be clear to describe its exact usage. Naming it as `read_proc_fdinfo` may be better.